kimchi/src/circuits/lookup/index.rs

```diff
@@ -36,6 +36,108 @@ pub enum LookupError {
     TableIDZeroMustHaveZeroEntry,
 }
 
+#[derive(Clone, Serialize, Deserialize, Debug, Default)]
+pub struct LookupSelectors<T> {
+    pub chacha: Option<T>,
+    pub chacha_final: Option<T>,
+    pub lookup_gate: Option<T>,
```

Comment:
not sure I understand why this struct is so generic. Why not make `T` -> `Evaluations<F, D<F>>` ?

Reply:
also all of these fields are `pub` and missing doc

Reply:
I wouldn't have added the generic parameter if it was always `Evaluations<F, D<F>>`. It's used with `PolyComm<G>` in `VerifierIndex`.